pkg/internal/network-conf-template.ps1

```diff
@@ -0,0 +1,81 @@
+# This script ensures the contents of the CNI config file is correct, and returns the HNS endpoint IP
```

Comment:
Shouldn't there be unit test for testing the generation from the teamplate?

Reply:
Added. Not sure of the value added when compared to the extra file required to do this, but I am fine with keeping it. PTAL and let me know what you think.